fluentui_topappbars/src/main/java/com/microsoft/fluentui/tokenized/AppBar.kt

```diff
@@ -79,6 +83,7 @@ fun AppBar(
     logo: @Composable (() -> Unit)? = null,
     searchMode: Boolean = false,
     navigationIcon: FluentIcon = FluentIcon(AppBarIcons.Arrowback, flipOnRtl = true),
+    navigationIconModifier: Modifier? = null,
```

Comment:
need not to expose navigationIconModifier

Reply:
okay, removed